src/components/CurrencyLogo/index.tsx

```diff
@@ -1,14 +1,16 @@
-import { Currency, ETHER, Token } from '@uniswap/sdk'
+import { Currency, ETHER as FUSE, Token, ChainId } from 'uniswap-fuse-sdk'
 import React, { useMemo } from 'react'
 import styled from 'styled-components'
 
+import FuseLogo from '../../assets/images/fuse-logo.png'
 import EthereumLogo from '../../assets/images/ethereum-logo.png'
 import useHttpLocations from '../../hooks/useHttpLocations'
 import { WrappedTokenInfo } from '../../state/lists/hooks'
 import Logo from '../Logo'
+import { useActiveWeb3React } from '../../hooks'
 
 const getTokenLogoURL = (address: string) =>
-  `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${address}/logo.png`
+  `https://raw.githubusercontent.com/mul53/default-token-list/master/assets/${address}/logo.png`
```

Comment:
Need to switch this for using our fuseio repo. If something missing there please open a PR

Reply:
https://www.npmjs.com/package/@fuseio/fuse-swap-default-token-list

Reply:
why don't we use trust wallet instead? I can open a pr there